railstats.py

```
@@ -64,25 +64,30 @@
         else:
             counts['station_visits'][row['destination']] = 1
 
+        if row['act_delay'] == None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)